### PR TITLE
Bump versions to v0.3.5

### PR DESCRIPTION
**Pull request by sea212**
Also updates *changelog_for_devs.md*

### CODE REVIEW

**Comment by sea212** [comment]
Reviewers, can you please also verify that *changelog_for_devs.md* is complete?

**Comment by saboonikhil** [comment]
> Reviewers, can you please also verify that _changelog_for_devs.md_ is complete?

Is `swap_fee` part of `v0.3.5` release according to https://github.com/zeitgeistpm/zeitgeist/milestone/9 ?

**Comment by maltekliemann** [comment]
@saboonikhil Yeah, that's an error. Swap fees were supposed to be in 0.3.4 but apparently didn't make it. Yes, they will be part of 0.3.5.

**Review by maltekliemann — APPROVED** [review]
Yeah, looks good to me! :thumbsup:

File: docs/changelog_for_devs.md [hunk 1]
```diff
@@ -17,6 +19,8 @@
   Furthermore, there's a maximum swap fee, specified by the `swaps` pallet's
   on-chain constant `MaxSwapFee`.
```
Comment (Chralt98) [inline docs/changelog_for_devs.md:20]: Isn't the addition of `zrml-styx` worth to mention? https://github.com/zeitgeistpm/zeitgeist/commit/1307e2a78b96d2603aca3497334132b62ebcc2de

Or has this file `changelog_for_devs` just the aim to inform the front end when something changes, which is already existing?

**Review by Chralt98 — APPROVED** [review]
- [X] Cargo.toml's updated
-  Battery-Station and the Main-Net
    - [X] `spec_version` runtime version updated
    - [X] `transaction_version` updated, because the dispatch parameters changed in https://github.com/zeitgeistpm/zeitgeist/commit/852ebea1ac6a3d5d3ec6212e01fa2b588a27a65c

File: docs/changelog_for_devs.md (hunk 1 above)
Comment (sea212) [reply docs/changelog_for_devs.md:20]: You are right, we should include it. Added in [7783b04](https://github.com/zeitgeistpm/zeitgeist/pull/748/commits/7783b046196c9c8923c6dd53a150bb72fd20bb1f).

**Pull request merged**